R/read_model_out_file.R

```diff
@@ -1,16 +1,25 @@
 #' Read a model output file
 #'
 #' @inheritParams check_valid_round_id
+#' @inheritParams hubData::create_hub_schema
 #' @param coerce_types character. What to coerce column types to on read.
 #' - `hub`: read in (`csv`) or coerce (`parquet`, `arrow`) to hub schema.
```

Comment:
Should we had that it's the default selection?

Reply:
Sure. Fixed in 3e9b0f4c0b526e34a59516a4e7222d1fb9059bd3